### DISCUSSION TITLE
Custom console log formatters

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#3983] [Enhancement Queue]
Allow people to customize the default output that Eleventy logs to console.

This could be used to log the gzip compressed size of a template, a la:
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/39355/46806540-741bc700-cd2d-11e8-9fd7-85fe60af0d17.png">

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

The enhancement backlog can be found here: https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+

Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
A vote to disable `[11ty]` prefix for folks that already have a line prefix: https://mas.to/@jamesdoc/115745634195806806

<img width="1330" height="638" alt="image" src="https://github.com/user-attachments/assets/4c1b7d25-bb2b-4777-a51f-ef87971da31f" />

**Reply by jamesdoc** [reply]
Thanks for pulling this in Zach - for future reference this is coming in thanks to [`concurrently`](https://www.npmjs.com/package/concurrently).
